README.md

```diff
@@ -30,38 +30,100 @@ docker login
+Data Directory:                       /Users/username/Library/Application Support/kurtosis/engine-data/enclaves/kt2022-04-04t17.17.03.135
+Enclave Status:                       EnclaveContainersStatus_RUNNING
+API Container Status:                 EnclaveAPIContainerStatus_RUNNING
+API Container Host GRPC Port:         127.0.0.1:55783
+API Container Host GRPC Proxy Port:   127.0.0.1:55784
+
+========================================= Kurtosis Modules =========================================
+GUID   LocalPortBindings
+
+========================================== User Services ==========================================
+GUID   ID   LocalPortBindings
+```
+
+If you pain attention to the screen you will se that we have the enclave but no user service or Kurtosis module has been created yet
+
+Now that we have an enclave, let's put something in it!, go ahead and execute the following command to run a `webserver` (based on the httpd Docker image) inside the enclave:
+
+```
+kurtosis service add demo webserver httpd --ports http=80
+```
+
+And the screen will show us something like this:
+
+```
+Service ID: webserver
+Ports Bindings:
+   http:   80/tcp -> 127.0.0.1:63825
+```
+
+Now, you can go to your browser and check that the webserver is up and running through the local url provided in the http port binding (e.g.: 127.0.0.1:63825)
+
+Let's inspect the enclave again:
+
+```
+kurtosis enclave inspect demo
+```
+
+Now, you should see the webserver listed in the user service slot, like this example:
 
-```javascript
-getServicesResult = await enclaveCtx.getServices()
-services = getServicesResult.value
 ```
+========================================== User Services ==========================================
+GUID                   ID          LocalPortBindings
+webserver-1649186256   webserver   80/tcp -> 127.0.0.1:63825
+```
+
+Finally, if you don't want to continue working with the service you can ger rid of it doing the following:
 
-We haven't started any services yet, so the enclave will be empty. Note how we called `await` on `enclaveCtx.getServices()`. This is because every `enclaveCtx` call is asynchronous and returns a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise); `await`ing blocks until that value is available.
+```
+kurtosis service rm demo webserver
+```
 
 
 Step Three: Start An Ethereum Network (5 minutes)
 -------------------------------------------------
-Now that we have an enclave, let's put something in it! Ethereum is one of the most popular blockchains in the world, so let's get a private Ethereum network running:
+Now that we know how to create and enclave and add a service, we can go one step ahead and start enjoying one of the most important features Kurtosis have, the Kurtosis modules.
+
+Ethereum is one of the most popular blockchains in the world, so let's get a private Ethereum network running just to executing a Kurtosis module:
```

Comment:
again really like the ideas; refining further:
```
Ethereum is the most popular smart contract blockchain in the world, so let's create a private Ethereum network in Kurtosis:
```
(I left out the "executing a Kurtosis module" part because we explain it later)